### DISCUSSION TITLE
Apple CarPlay Dashboard Support

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#4164] [Ideas]
Hello, I use OM a lot as a second option to Waze, however I miss some integrations, such as support for the Apple CarPlay Dashboard.

I wanted to know if there is already any project to support this function in the future. 

Thanks!

![BD0DD69A-9084-471E-8EB4-4E021B0B7D42](https://user-images.githubusercontent.com/41340982/209866869-0dd05e41-4fe6-4a88-869f-ec046e42b896.jpg)

**Comment by davidcollini** [comment]
Does Waze have this integration, or is it just Apple Maps?

**Comment by biodranik** [comment]
Is it technically feasible for non-Apple apps?

**Reply by ghost** [reply]
Yes, some navigation apps have this feature, such as Waze and Google Maps.

I saw this documentation at Apple itself, I believe it's not that complicated: https://developer.apple.com/documentation/carplay/displaying_content_in_carplay

**Reply by scottmcqueen** [reply]
Carplay is also used by apps like Gaia, if this could be upped in priority it would be amazing, even motorcycles have Carplay and Android Auto now.
